packages/peregrine/lib/targets/peregrine-intercept.js

```diff
@@ -24,17 +41,27 @@ function TalonWrapperConfig(addTransforms) {
     });
     return {
         /**
-         * @memberof TalonWrapperConfig
+         * Provides access to the talon used in Venia's ProductFullDetail UI component.
```

Comment:
In general, I think `component` is sufficient.

```suggestion
         * Provides access to the talon used in Venia's ProductFullDetail component.
```

Reply:
Updated